### PR TITLE
fix some translation errors in Chinese

### PR DESCRIPTION
**Pull request by biaoma-ty**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)
- Non-significant (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

...

Detailed description / Documentation draft:

...

By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

### CODE REVIEW

**Comment by blinkov** [comment]
@f7753 actually, those look like the function names and they're supposed to be all in English as they are used in SQL queries.

File: docs/zh/sql-reference/functions/date-time-functions.md [hunk 1]
```diff
@@ -18,19 +18,19 @@ SELECT
 
 仅支持与UTC相差一整小时的时区。
 
-## toTimeZone {#totimezone}
+## 转换为时区 {#totimezone}
```
Comment (hczhcz) [inline docs/zh/sql-reference/functions/date-time-functions.md:21]:
```suggestion
## toTimeZone {#totimezone}
```

File: docs/zh/sql-reference/functions/date-time-functions.md [hunk 2]
```diff
@@ -18,19 +18,19 @@ SELECT
...
 
 将Date或DateTime转换为指定的时区。
 
-## 玩一年 {#toyear}
+## 转换为年 {#toyear}
```
Comment (hczhcz) [inline docs/zh/sql-reference/functions/date-time-functions.md:25]:
```suggestion
## toYear {#toyear}
```

File: docs/zh/sql-reference/functions/date-time-functions.md [hunk 3]
```diff
@@ -18,19 +18,19 @@ SELECT
...
 
 将Date或DateTime转换为包含年份编号（AD）的UInt16类型的数字。
 
-## 到四分钟 {#toquarter}
+## 转换为季度 {#toquarter}
```
Comment (hczhcz) [inline docs/zh/sql-reference/functions/date-time-functions.md:29]:
```suggestion
## toQuarter {#toquarter}
```

File: docs/zh/sql-reference/functions/date-time-functions.md [hunk 4]
```diff
@@ -18,19 +18,19 @@ SELECT
...
 
 将Date或DateTime转换为包含季度编号的UInt8类型的数字。
 
-## toMonth {#tomonth}
+## 转换为月份 {#tomonth}
```
Comment (hczhcz) [inline docs/zh/sql-reference/functions/date-time-functions.md:33]:
```suggestion
## toMonth {#tomonth}
```

**Review by hczhcz — COMMENTED** [review]
Translating a function name does not make sense.

BTW, the machine translator translates "toYear" into "playing for one year".. interesting one.

**Comment by alexey-milovidov** [comment]
@blinkov We have to manually revert translation of all function names
and also to fix machine translation script.

**Comment by blinkov** [comment]
@alexey-milovidov yes, there's some room for improvement. Keywords, function names, and other stuff that's supposed to be part of a query or config need to be marked as such in English source (usually by backticks, but probably headers need some alternative). It's already been done in a significant part of the content, but there are still some parts where they are historically indistinguishable from plain text.

**Pull request merged**